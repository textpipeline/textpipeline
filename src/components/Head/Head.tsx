import React from 'react';
import { Helmet } from 'react-helmet';

const siteName = 'Text Pipeline';
const description = '';

const Head: React.FC = () => (
  <Helmet>
    <title>{siteName}</title>
    <meta charSet="utf-8" />
    <meta name="viewport" content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no" />
    <meta name="description" content={description} />
  </Helmet>
);

export default Head;
