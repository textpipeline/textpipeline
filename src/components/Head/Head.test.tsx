import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import Head from './Head';

describe('Head', () => {
  let rendered: ShallowWrapper;

  beforeEach(() => {
    rendered = shallow(<Head />);
  });

  it('renders correctly', () => {
    expect(rendered).toMatchSnapshot();
  });
});
