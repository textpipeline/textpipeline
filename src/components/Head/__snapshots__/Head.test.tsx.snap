// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Head renders correctly 1`] = `
<HelmetWrapper
  defer={true}
  encodeSpecialCharacters={true}
>
  <title>
    Text Pipeline
  </title>
  <meta
    charSet="utf-8"
  />
  <meta
    content="minimum-scale=1, initial-scale=1, width=device-width, shrink-to-fit=no"
    name="viewport"
  />
  <meta
    content=""
    name="description"
  />
</HelmetWrapper>
`;
